gradle-jdks-integration-tests/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'java'
+apply plugin: 'org.unbroken-dome.test-sets'
+
+testSets {
+    jdkIntegrationTest
```

Comment:
Not sure we need to call this `jdkIntegrationTest` (the whole project name has `jdk` in it), we normally just call this `integrationTest`.